README.md

```diff
@@ -0,0 +1,224 @@
+
+The content inside the website is fictional, as the photos of Speakers, Partners, and, etc. are randomly obtained through [Google Images](https://images.google.com/). The project was created using HTML, CSS, and JavaScript in order to meet [the requirements defined](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md) within the scope of the [Microverse Capstone Project](https://github.com/microverseinc/curriculum-professional-skills/blob/main/becoming-a-remote-professional/capstone-projects-at-microverse.md). More specially, I would like to point out that the website I create is done from scratch, but just to make it similar according to original design by Cindy Shin on Behance. See [Acknowledgements](#acknowledgements) for more details. 
+
+Capstone Project (Module 1) is built as part of the 12-month Microverse Full-Stack Web Development Program. The mobile project was first applied in this project, followed by the desktop one. 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+This capstone project is built and configured using the following tools:
+
+<details>
+  <summary>Languages</summary>
+  <ul>
+    <li><a href="https://html.com/">HTML</a></li>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+    <li><a href="https://www.javascript.com/">JavaScript</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Development Applications</summary>
+  <ul>
+    <li><a href="https://code.visualstudio.com/">Visual Studio Code</a></li>
+    <li><a href="https://getbootstrap.com/">Bootstrap</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Git / GitHub</summary>
+  <ul>
+    <li><a href="https://git-scm.com/">Git</a></li>
+    <li><a href="https://www.github.com/">Github</a></li>
+    <li><a href="https://pages.github.com/">Github Pages</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Web Configuration Applications</summary>
+  <ul>
+    <li><a href="https://developer.chrome.com/docs/lighthouse/overview/">Lighthouse</a></li>
+    <li><a href="https://stylelint.io/">Stylelint</a></li>
+    <li><a href="https://webhint.io/">Webhint</a></li>
+    <li><a href="https://eslint.org/">ES Lint</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+### Key Features <a name="key-features"></a>
+
+The three key features used in my personal portfolio project include: 
+
+- An interactive mobile menu page.
+- An interactive photo gallery about laxACT's previous streaming conferences.
+- Storing the details of conference speakers through 1 separate JS file.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+## 🚀 Live Demo <a name="live-demo"></a>
+
+The Live Demo of this Capstone Project is available now on GitHub Pages and Loom.
+
+- [Live Demo Link on GitHub pages](https://katarighe.github.io/microverse-capstone-project-1/)
```

Comment:
- [ ] @katarighe  Great work implementing the changes requested however the images on the speaker section are not showing on the live demo, kindly consider looking into the issue 
![image](https://github.com/katarighe/microverse-capstone-project-1/assets/61408860/94e8cb37-5602-4617-a09f-1b50a25935c5)